tests/backends/tests.py

```diff
@@ -745,6 +745,19 @@ def test_cursor_contextmanager_closing(self):
             self.assertIsInstance(cursor, CursorWrapper)
         self.assertTrue(cursor.closed)
 
+    @skipUnlessDBFeature('test_db_allows_multiple_connections')
```

Comment:
Perhaps I am wrong but here is my thoughts : 
testing that `is_usable` does not raise is fine but not enought to me. I think we need to ensure - a non-existing connection is unusable
-  an open one is 
- and a closed one is not.

For the sqlite3 case the connection will never close (it can't) so the test will fail. The bug can only be reproduced with `postgresql` `oracle` and `mysql` so testing sqlite is pointless to me.

What we can do is the following:
- put the logic in base
- run the test for the "test_db_allows_multiple_connections" (only databases with this feature can have this bug)

What do you think @cjerdonek ?